### DISCUSSION TITLE
how did s-pscripts get write access

### DISCUSSION THREAD

**Discussion by schoolexploitkid** [3kh0/ext-remover#1303] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
how

**Comment by awsome-webdev** [comment]
maybe he deserved it:
literally the most helpful person in the repo
![image](https://github.com/user-attachments/assets/70e573db-303a-4b42-ba7a-45fbc8b45881)

**Reply by schoolexploitkid** [reply]
bro he did that after he got write access

**Reply by awsome-webdev** [reply]
he was still helpful....
why don't you want pscripts to have write access

**Reply by akabutnicer** [reply]
real

**Reply by akabutnicer** [reply]
used to be me throwing up these numbers before I stopped caring as much

**Reply by S-PScripts** [reply]
> maybe he deserved it: literally the most helpful person in the repo ![image](https://private-user-images.githubusercontent.com/126301230/388731541-70e573db-303a-4b42-ba7a-45fbc8b45881.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzIyNzkzMjIsIm5iZiI6MTczMjI3OTAyMiwicGF0aCI6Ii8xMjYzMDEyMzAvMzg4NzMxNTQxLTcwZTU3M2RiLTMwM2EtNGI0Mi1iYTdhLTQ1ZmJjOGI0NTg4MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyMlQxMjM3MDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02MzEyYTA5NjEzNzU1NjQ2MzQyZjA4YWQ2NDAyN2M1YWIxOThiMzc4ODU1MjBiMGIyZDNiNmE3YTI2MjFhNjVkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.e2iNAuyi6NKZ1ldlKfuGGw-oj4Kk8TSCUeQbGXqIZtc)

nah 3kh0 gave it to me before i marked those answers
i just answered dead topics 
im not sure why 3kh0 gave me access xd, maybe because i often comment in discussions?

**Reply by Luigi-Master** [reply]
Lucky! 😒

**Reply by awsome-webdev** [reply]
> > maybe he deserved it: literally the most helpful person in the repo ![image](https://private-user-images.githubusercontent.com/126301230/388731541-70e573db-303a-4b42-ba7a-45fbc8b45881.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzIyNzkzMjIsIm5iZiI6MTczMjI3OTAyMiwicGF0aCI6Ii8xMjYzMDEyMzAvMzg4NzMxNTQxLTcwZTU3M2RiLTMwM2EtNGI0Mi1iYTdhLTQ1ZmJjOGI0NTg4MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyMlQxMjM3MDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02MzEyYTA5NjEzNzU1NjQ2MzQyZjA4YWQ2NDAyN2M1YWIxOThiMzc4ODU1MjBiMGIyZDNiNmE3YTI2MjFhNjVkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.e2iNAuyi6NKZ1ldlKfuGGw-oj4Kk8TSCUeQbGXqIZtc)
> 
> nah 3kh0 gave it to me before i marked those answers i just answered dead topics im not sure why 3kh0 gave me access xd, maybe because i often comment in discussions?

maybe idk 🤔
